src/Exception/ApplicationException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SnowflakeTransformation\Exception;
+
+class ApplicationException extends \Exception
```

Comment:
https://github.com/keboola/common-exceptions

Reply:
je to tam